test/javascripts/acceptance/assigned-topic-test.js

```diff
@@ -123,6 +123,16 @@ acceptance("Discourse Assign | Assigned topic", function (needs) {
       "shows reassign dropdown at the bottom of the topic"
     );
   });
+
+  test("Public user cannot see footer button", async (assert) => {
```

Comment:
`Public` is a little vague and is not a common term that we use here. Perhaps something like `Footer button not displayed for user without the ability to assign`?

Reply:
I only used `Public` because the user setting says "public assigns" and the hint is "Allow general public to see..."